src/modules/services_manager_service.rb

```diff
@@ -8,186 +10,31 @@ module Yast
 
   class ServicesManagerServiceClass < Module
     include Yast::Logger
+    extend Yast::I18n
 
-    LIST_UNIT_FILES_COMMAND = 'systemctl list-unit-files --type service'
-    LIST_UNITS_COMMAND      = 'systemctl list-units --all --type service'
-    STATUS_COMMAND          = 'systemctl status'
-    # FIXME: duplicated in Yast::Systemctl
-    COMMAND_OPTIONS         = ' --no-legend --no-pager --no-ask-password '
-    TERM_OPTIONS            = ' LANG=C TERM=dumb COLUMNS=1024 '
-    SERVICE_SUFFIX          = '.service'
+    SERVICE_SUFFIX = '.service'
+
+    START_MODE = {
+      on_boot:   N_('On Boot'),
+      on_demand: N_('On Demand'),
```

Comment:
I think this need some translators hint and same with manual as it is quite context dependent.